src/node-cron.ts

```diff
@@ -0,0 +1,10 @@
+import { handleNodeCheck } from './handlers';
+
+if (process.mainModule === module) {
+  handleNodeCheck()
+    .catch((err) => {
+      console.log('Node Cron Failed');
+      console.error(err);
+      process.exit(1);
+    });
+}
```

Comment:
any particular reason not to handle node & chromium on the same cron job?

Reply:
I started with a separate file because I was testing a lot locally by running `node node-cron.ts`.

Now that you mention it, it would make a lot of sense to merge together for production!

Reply:
we can have both separated files that are testable locally and a single cron job that runs on heroku. have the separate files be executable by themselves and the cron job should just call both functions :)

Reply:
Ultimately refactored so we don't need multiple files! Testing locally would be easy enough... just need to remove the handlers from the array. :)